parsers/einpark/derbypub.ts

```diff
@@ -20,7 +20,7 @@ export class DerbyPub implements IParser {
             }
         });
 
-        doneCallback(dayMenu);
+        return Promise.resolve(dayMenu);
```

Comment:
This is the only unfortunate thing about the promisification, parsers has to return `Promise` even though they does nothing asynchronously.
It is due to Skolka parser that needs to do extra call (OCR) which is async operation. But on the other side, we never know when also the whole parsing (Cheerio) will use non-blocking operations (Promises).
Honestly, I was surprised when I learned parser like Cheerio is not using non-blocking API (Promises).